client/app/account/billing/payment/method/add/translations/Messages_fr_FR.xml

```diff
@@ -9,6 +9,7 @@
    <translation id="billing_payment_method_add_selected_paypal_redirect" qtlid="517004">En cliquant sur "Ajouter", vous allez être redirigé vers la plateforme PAYPAL.</translation>
    <translation id="billing_payment_method_add_selected_original_credit_card_redirect" qtlid="517017">En cliquant sur le bouton "Ajouter", un document sera téléchargé. Veuillez suivre la procédure décrite dans le document pour valider le moyen de paiement.</translation>
    <translation id="billing_payment_method_add_credit_card_selected" qtlid="351461">Votre carte bancaire sera activée immédiatement.</translation>
+   <translation id="billing_payment_method_add_credit_card_selected_explain">Pour enregistrer une nouvelle carte bancaire, une empreinte d’1€ est faite auprès de votre organisme bancaire afin de valider la saisie et la validité de votre carte. Le montant d’1€ ne sera pas prélevé.</translation>
```

Comment:
The amount should be a variable :)

No gain to translate it, and, if one day the amount is dynamic, no need to translate it again :)